installers/install.ps1

```diff
@@ -63,9 +63,9 @@ function isAdmin
 
 function promptYN([string]$msg)
 {
-    $response = Read-Host -Prompt $msg" [y/N]`n"
+    $response = Read-Host -Prompt $msg" [Y/n]`y"
```

Comment:
```suggestion
    $response = Read-Host -Prompt $msg" [Y/n]`n"
```

I think that was a line ending

Reply:
Fixed.

Reply:
Hold on, that commit isn't showing up here.

Reply:
Ok fixed -- weird GitHub issue. I had to push a new commit to get the other one to show up here.